release_docs/RELEASE.txt

```diff
@@ -778,7 +781,7 @@ CMake vs. Autotools installations
 =================================
 While both build systems produce similar results, there are differences.
 Each system produces the same set of folders on linux (only CMake works
-on standard Windows); bin, include, lib and share. Autotools places the
+on standard Windows); bin, include, lib, and share. Autotools places the
```

Comment:
I agree with the intention but I was informed a decade ago that there is not a comma before an and in a list of items.